packages/react-core/src/Authenticator/hooks/types.ts

```diff
@@ -75,14 +75,14 @@ export type CommonRouteProps = {
   handleBlur: UseAuthenticator['updateBlur'];
   handleChange: UseAuthenticator['updateForm'];
   handleSubmit: UseAuthenticator['submitForm'];
+  validationErrors?: UseAuthenticator['validationErrors'];
 };
 
 /**
  * Base Route component props
  */
 export type ConfirmResetPasswordBaseProps<FieldType = {}> = {
   resendCode: UseAuthenticator['resendCode'];
-  validationErrors?: UseAuthenticator['validationErrors'];
```

Comment:
Added to common props as opposed to each individual Base Route component props (same comment for below components).